resources/views/community/components/user/profile/developer-claims-list.blade.php

```diff
@@ -0,0 +1,29 @@
+@props([
+    'claims' => [],
+    'label' => 'Label',
+    'numAllowedClaims' => null, // ?int
+])
+
+<div>
+    <p class="text-2xs mb-1">
+        <span class="font-bold">{{ $label }}</span>
+
+        @if ($numAllowedClaims !== null && count($claims) <= $numAllowedClaims)
+            ({{ count($claims) }} of {{ $numAllowedClaims }} allowed)
+        @endif
```

Comment:
why is this restricted when `claims` exceed `numAllowedClaims`? Wouldn't it be better to show "2 of 1 allowed" instead of not showing anything?

Reply:
Removed `count($claims) <= $numAllowedClaims` from the conditional.